products/cli/project-commands/helper-commands.md

```diff
@@ -50,6 +50,20 @@ To exclude specific:
 shopware-cli project admin-watch --skip-extensions <name>,<second>....
 ```
 
+### Building only custom extensions
+
+When working with a lot of 3rd party extensions, `project storefront-build` and `project admin-build` would become slow, when all extensions are built.
+This is unnecessary, because store extensions are shipped together withtheir assets.
```

Comment:
withtheir is an typo see pipeline failure :)

Reply:
yep, fixed - please squash when merging